source/message/OP_MSG.rst

```diff
@@ -206,6 +215,40 @@ connection. All other errors during processing will be silently dropped, and
 will not result in the connection being closed.
 
 
+moreToCome On Responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When the ``moreToCome`` flag is set on a response it signals to the recipient
+that the sender will send additional responses on the connection. The recipient
+MUST continue to read responses until it reads a response with the ``moreToCome``
+flag not set, and MUST NOT send any more requests on this connection until
+it reads a response with the ``moreToCome`` flag not set. The client MUST
+either consume all messages with ``moreToCome`` flag set or close the connection.
+
+When the server sends responses with ``moreToCome`` flag set, it behaves as if
+the client sent a ``getMore`` request for the returned cursor for each of the
```

Comment:
This is the first mention of a cursor in the spec, and it's introduced here a bit awkwardly.  Consider restructuring so that it's clear that setting exhaustAllowed on the request is supported for command that return cursors, and that moreToCome responses are generally limited to implicit getMore requests from the client

Reply:
Where was it specified that exhaustallowed is only supported for commands that return cursors?

Reply:
I removed the cursor/getMore language from this section entirely.